Source/WebCore/Modules/webxr/NavigatorWebXR.h

```diff
@@ -40,6 +40,7 @@ class NavigatorWebXR final : public Supplement<Navigator> {
     WTF_MAKE_FAST_ALLOCATED;
 public:
     WEBCORE_EXPORT static WebXRSystem& xr(Navigator&);
+    WEBCORE_EXPORT static RefPtr<WebXRSystem> xrIfExists(Navigator&);
```

Comment:
This should return a `WebXRSystem*`, not a `RefPtr<WebXRSystem>`.

Reply:
This is what happens when I read your document.

Reply:
Seems like I need to clarify the document then :) None of the guidelines say to do this.